tests/compute/subresources/pausing.go

```diff
@@ -70,48 +70,6 @@ var _ = compute.SIGDescribe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com
 			Eventually(matcher.ThisVM(vm)).WithTimeout(300 * time.Second).WithPolling(time.Second).Should(matcher.BeReady())
 		})
 
-		It("[test_id:4598]should signal paused state with condition", func() {
```

Comment:
While this is unit testable, it is also expected behavior from use POV so please leave a test that demonstrate this